e2e/test/scenarios/joins/reproductions/29795-native-query-join-question-misses-metadata.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  restore,
+  visualize,
+  popover,
+  openOrdersTable,
+} from "e2e/support/helpers";
+
+describe.skip("issue 29795", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should allow join based on native query (metabase#29795)", () => {
+    const NATIVE_QUESTION = "native question";
+    const LIMIT = 5;
+    cy.createNativeQuestion(
+      {
+        name: NATIVE_QUESTION,
+        native: { query: `SELECT * FROM "PUBLIC"."ORDERS" LIMIT ${LIMIT}` },
+      },
+      { loadMetadata: true },
+    );
+
+    openOrdersTable({ mode: "notebook" });
+
+    cy.button("Join data").click();
+
+    popover().within(() => {
+      cy.icon("chevronleft").click();
+      cy.findByText("Saved Questions").click();
+      cy.findByRole("menuitem", { name: NATIVE_QUESTION }).click();
+    });
+
+    popover().within(() => {
+      cy.findByRole("option", { name: "ID" }).click();
+    });
+
+    popover().within(() => {
+      cy.findByRole("option", { name: "USER_ID" }).click();
+    });
+
+    visualize(response => {
+      expect(response.body.error).not.to.exist;
+      cy.findByText(`Showing ${LIMIT} rows`);
```

Comment:
This is better than not having any UI assertion but it still doesn't tell us did the visualization (probably table in this case) render.

Reply:
added another assert